src/Type/Definition/EnumType.php

```diff
@@ -80,6 +80,8 @@ public function getValues()
                         }
                     } else if (is_int($name) && is_string($value)) {
                         $value = ['name' => $value, 'value' => $value];
+                    } else if (is_int($name) && is_int($value)) {
+                        $value = ['name' => $value, 'value' => $value];                        
```

Comment:
This is not spec-compliant. Enum names must be alpha-numeric strings.

Reply:
Yes, too bad as I have some enum with numbers that should show up in GraphIQL as so, not something like _1, _2